spec/python_wrapper/smoke/doc_builder_value_spec.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+describe 'CDocBuilderValue tests', :critical do
```

Comment:
Since we use the critical tag to highlight the list of tests to run in [Action for the development team](
https://github.com/ONLYOFFICE/sdkjs/blob/aa764ca10e458758b4c18e1b0c89764731e504b3/.github/workflows/dev_check.yml#L97), we cannot unilaterally add tests to the job

Ilya Kirilov is against running a large number of tests on sdkjs, so we need to remove the tag and run a full run on our side